docs/reference/transform/apis/get-transform-stats.asciidoc

```diff
@@ -72,9 +72,81 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=size-transforms]
-  sorted by the `id` value in ascending order.
+The API returns an array of statistics objects for {transforms}, which are
+sorted by the `id` value in ascending order. All of these properties are
+informational; you cannot update their values.
+
+`checkpointing`::
+(object) Contains statistics about <<transform-checkpoints,checkpoints>>.
+`checkpointing`.`changes_last_detected_at`:::
+(date) The timestamp when changes were last detected in the source indices.
+`checkpointing`.`last`:::
+(object) Contains statistics about the last completed checkpoint. 
+`checkpointing`.`last`.`checkpoint`::::
+(TBD) A unique identifier for the checkpoint.
+`checkpointing`.`last`.`time_upper_bound_millis`::::
+(date) TBD
+`checkpointing`.`last`.`timestamp_millis`::::
+(date) TBD
+
+`id`::
+(string)
+include::{docdir}/rest-api/common-parms.asciidoc[tag=transform-id]
+
+`node`::
+(object) For started {transforms} only, the node upon which the {transform} is
+started.
+`node`.`attributes`:::
+(object) A list of attributes for the node.
+`node`.`ephemeral_id`:::
+(string) The node ephemeral ID.
+`node`.`id`:::
+(string) The unique identifier of the node. For example, "0-o0tOoRTwKFZifatTWKNw".
+`node`.`name`:::
+(string) The node name. For example, `0-o0tOo`.
+`node`.`transport_address`:::
+(string) The host and port where transport HTTP connections are accepted. For
+example, `127.0.0.1:9300`.
+`state`::
+(string) The status of the {transform}, which can be one of the following values:
++
+--
+* `indexing`: The {transform} is actively processing data and creating new
+documents.
+* `started`: The {transform} is running but not actively indexing data.
+* `stopped`: The {transform} is stopped.
+--
+
+`stats`::
+(object) An object that provides statistical information about the {transform}.
+`stats`.`documents_indexed`:::
+(TBD) The number of new documents that have been indexed.
+`stats`.`documents_processed`:::
+(TBD) The number of documents that have been processed.
+`stats`.`exponential_avg_checkpoint_duration_ms`:::
+(double) Exponential moving average of the duration of the checkpoint, in milliseconds.
+`stats`.`exponential_avg_documents_indexed`:::
+(double) Exponential moving average of the number of new documents that have been
+indexed.
+`stats`.`exponential_avg_documents_processed`:::
+(double) Exponential moving average of the number of documents that have been
+processed.
+`stats`.`index_failures`:::
+(long) The number of indexing failures.
+`stats`.`index_time_in_ms`:::
+(long) The amount of time spent indexing, in milliseconds.
+`stats`.`index_total`:::
+(long) The number of indices created.
+`stats`.`pages_processed`:::
+(TBD) The number of pages processed.
+`stats`.`search_failures`:::
+(long) The number of search failures. 
+`stats`.`search_time_in_ms`:::
+(long) The amount of time spent searching, in milliseconds.
+`stats`.`search_total`:::
+(long) TBD 
+`stats`.`trigger_count`:::
+(TBD) TBD
```

Comment:
The number of times the transform has been triggered by the scheduler.

(The scheduler triggers the transform indexer to e.g. check for updates / ingest new data, this can be controlled via the `frequency` parameter in the config: https://www.elastic.co/guide/en/elasticsearch/reference/master/put-transform.html#put-transform-request-body)